contracts/SportMarkets/Rundown/TherundownConsumer.sol

```diff
@@ -504,7 +504,6 @@ contract TherundownConsumer is Initializable, ProxyOwned, ProxyPausable {
                 _gameId,
                 string(abi.encodePacked(game.homeTeam, " vs ", game.awayTeam)), // gameLabel
                 game.startTime, //maturity
-                0, //initialMint
```

Comment:
I would leave this to not have to release all these contracts

Reply:
done.